Cubical/Foundations/EquivCharacterization.agda

```diff
@@ -0,0 +1,61 @@
+{-
+
+More theory about equivalences/different characterizations of equivalence
+
+- 'cong f' is an equivalence, if f is an equivalence
+
+-}
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Foundations.EquivCharacterization where
+
+open import Cubical.Core.Everything
+
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Equiv
+
+isEquivCong : ∀ {ℓ} {A B : Set ℓ} {x y : A} (e : A ≃ B) → isEquiv (λ (p : x ≡ y) → (cong (e .fst) p))
+isEquivCong e = EquivJ (λ (B' A' : Set _) (e' : A' ≃ B') →
+                         (x' y' : A') → isEquiv (λ (p : x' ≡ y') → cong (e' .fst) p))
+                       (λ _ x' y' → idIsEquiv (x' ≡ y')) _ _ e _ _
+
+congEquiv : ∀ {ℓ} {A B : Set ℓ} {x y : A} (e : A ≃ B) → (x ≡ y) ≃ (e .fst x ≡ e .fst y)
+congEquiv e = ((λ (p : _ ≡ _) → cong (e .fst) p) , isEquivCong e)
```

Comment:
Could you please move this to the Equiv file? I think it makes more sense to have it there instead of in this file.

Reply:
It uses univalences, so that would introduce cyclic dependencies.

Reply:
I see... That's unfortunate. I wonder how we could reorganize things to avoid this

Reply:
Maybe introducing a file on isomorphisms could help this. It would contain the proof that iso implies equiv which is used in Univalence. This would make the definition not be cyclic as the definition of equivalence is exported by the Core.

Reply:
Did it. But I had to move things like 'compEquiv' to the new 'Isomorphism...'

Reply:
I don't see why... In my head the dependency should be as follows:

1. Foundations.Isomorphism: section/retract, Iso, isoToEquiv

2. Foundations.Equiv: depends on 1 and contains all the theory about equivalences

3. Foundations.Univalence: depends on 1 and 2

4. Foundations.PathSplitEquiv: depends on 1-3 and contains in particular congEquiv.

This should not lead to any circularities? The fact that congEquiv ends up in 4 is a bit unfortunate, but we can add a comment to 2 with a pointer to where it can be found.